docs/api.rst

```diff
@@ -262,4 +263,4 @@ Exceptions
     PCMSettingError
     InvalidDatasetError
     DatasetRegisterError
-    RecordTypeError
\ No newline at end of file
+    RecordTypeError
```

Comment:
Emacs kept trying to add this newline to the end of the file each time I edited it. I finally forgot to revert the change before committing, hopefully that's okay.